deploy/opencraft.yml

```diff
@@ -0,0 +1,11 @@
+- name: Deploy the OpenCraft Instance Manager
+  hosts: all
+  become: true
+  roles:
+    - name: common-server
+      COMMON_SERVER_NO_BACKUPS: true    # The instance manager VM is stateless
+      FORWARD_MAIL_MYDOMAIN: "{{ inventory_hostname }}"
```

Comment:
Thanks.  I didn't update my repos with the lates changes yet to reduce the number of times I have to do that.